generics/services/improvement-project.js

```diff
@@ -323,11 +323,112 @@ const projectDocuments = function (
+            
+            function improvementProjectCallback(err, data) {
+                
+                let result = {
+                    success : true
+                };
+
+                if (err) {
+                    
+                    result.success = false;
+                } else {
+
+                    let response = data.body;
+                    if( response.status === httpStatusCode['ok'].status) {
+                        result["data"] = response.result;
+                    } else {
+                        result.success = false;
+                    }
+                }
+                
+                return resolve(result);
+            }
+
+            const options = {
+                headers : {
+                    "content-type": "application/json",
+                    "internal-access-token": process.env.INTERNAL_ACCESS_TOKEN,
+                    "x-authenticated-user-token" : userToken
+                },
+                json : {
+                }
+            };
+            
+            request.get(url,options,improvementProjectCallback);
+
+        } catch (error) {
+            
+            return reject(error);
+        }
+    })
+}
+const getCreatedProjectStats = function (
+    {userToken}
+) {
+    
+    return new Promise(async (resolve, reject) => {
+        try {
+            
+            const url = process.env.ML_PROJECT_SERVICE_URL + constants.endpoints.LIST_CREATED_STATS
+            
+            function improvementProjectCallback(err, data) {
+                
+                let result = {
+                    success : true
+                };
+
+                if (err) {
+                    
+                    result.success = false;
+                } else {
+
+                    let response = data.body;
+                    if( response.status === httpStatusCode['ok'].status) {
+                        result["data"] = response.result;
+                    } else {
+                        result.success = false;
+                    }
+                }
+                
+                return resolve(result);
+            }
+
+            const options = {
+                headers : {
+                    "content-type": "application/json",
+                    "internal-access-token": process.env.INTERNAL_ACCESS_TOKEN,
+                    "x-authenticated-user-token" : userToken
+                },
+                json : {
+                }
+            };
+            
+            request.get(url,options,improvementProjectCallback);
+
+        } catch (error) {
+            
+            return reject(error);
+        }
+    })
+}
+
 
 module.exports = {
     assignedProjects : assignedProjects,
     importedProjects : importedProjects,
     getProjectDetail : getProjectDetail,
     getTemplateDetail : getTemplateDetail,
-    projectDocuments : projectDocuments
+    projectDocuments : projectDocuments,
+    listProjectOverviewInfo,
```

Comment:
while exporting check how above function have be exported @borkarsaish65

Reply:
done